packages/circuits/tests/select-regex-reveal.test.ts

```diff
@@ -95,23 +99,7 @@ describe("Select Regex Reveal", () => {
         } catch (error) {
             expect((error as Error).message).toMatch("Assert Failed");
         }
-    });
 
-    it("should fail when startIndex is larger than max length", async function () {
```

Comment:
Wait why did we remove this test

Reply:
By our pattern we are constraining indices where it originates (main circuits). Adding them to each utils might be redundant as main circuits would be using multiple utils. We are not doing these checks on other circuits as well. This was added recently by another PR but we already have comments saying "assumes valid index"